addons/explosives/CfgVehicles.hpp

```diff
@@ -338,4 +338,80 @@ class CfgVehicles {
     class O_Soldier_sniper_base_F;
     class O_sniper_F: O_Soldier_sniper_base_F {MACRO_ADDCLAYMOREKIT};
     class O_spotter_F: O_Soldier_sniper_base_F {MACRO_ADDCLAYMOREKIT};
+
+    // Editor placed mines
+    class MineBase;
+    class SLAMDirectionalMine: MineBase {
+        displayName = CSTRING(Module_SLAMSideAttack_DisplayName);
+    };
+    class ACE_SLAMBottomMine: SLAMDirectionalMine {
+        author = ECSTRING(common,aceteam);
+        ammo = "ACE_SLAMDirectionalMine_Magnetic_Ammo";
+        displayName = CSTRING(Module_SLAMBottomAttack_DisplayName);
+        // TODO: Find a way to place the mine laying down instead of standing up
```

Comment:
It doesn't appear to, nor does the regular eventhandlers work as far as I can tell.